sycl/test-e2e/Matrix/joint_matrix_apply_two_matrices_impl.hpp

```diff
@@ -0,0 +1,147 @@
+//==------- joint_matrix_apply_two_matrices_impl.hpp  - DPC++
+// joint_matrix---==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+template <typename Tc, typename Ta, size_t M, size_t N>
+bool assert_ops_ref(Tc *C, Tc *D, Ta *A, Ta *Ar) {
```

Comment:
The function doesn't assert

```suggestion
bool verify(Tc *C, Tc *D, Ta *A, Ta *Ar) {
```